src/DssSpell.sol

```diff
@@ -16,24 +16,37 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
+interface StarknetBridgeLike {
+    function close() external;
+}
+
+interface StarknetGovRelayLike {
+    function relay(uint256 spell) external;
+}
+
+interface StarknetEscrowLike {
+    function approve(address token, address spender, uint256 value) external;
+}
+
+
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/57700b7c1eab15c213e5c92cd9ebf5de9df44f24/governance/votes/Executive%20vote%20-%20October%2019%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/27dc14fd37c03c152eca32296076f5603c9fd4db/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md -q -O - 2>/dev/null)"
```

Comment:
how did you get this url? The method I know to get it gave me: https://raw.githubusercontent.com/makerdao/community/9a78568330b6f34489639cd699cc4d00631ef650/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md (which gives the same hash, so just curious about the different approaches)

Reply:
1) Grabbed the [link](https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md) from new-spells
2) Clicked history and viewed the file (exec) at the latest commit ([this](https://github.com/makerdao/community/blob/27dc14fd37c03c152eca32296076f5603c9fd4db/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md) URL)
3) Hit 'Raw' and copied the URL


The result was:
https://raw.githubusercontent.com/makerdao/community/27dc14fd37c03c152eca32296076f5603c9fd4db/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md

Reply:
ok, great! thank you, that is probably better than what I did (though I thought it would be the same):

1. Go to that [link](https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md)
2. click 3 dots, click copy permalink
3. go to permalink
4. Click 'Raw' and copy url.

I suspect this method is giving us some sort of hash of the file whereas yours is a specific commit, which like i said is probably better

Reply:
@iamchrissmith yours is the last commit of the PR and Arby's one is the merge commit.